pkg/controller/daemon/daemoncontroller.go

```
@@ -521,21 +527,18 @@ func (dsc *DaemonSetsController) manage(ds *extensions.DaemonSet) error {
 	errCh := make(chan error, createDiff+deleteDiff)
 
 	glog.V(4).Infof("Nodes needing daemon pods for daemon set %s: %+v, creating %d", ds.Name, nodesNeedingDaemonPods, createDiff)
+	podtemplate, err := daemonutil.GetOrCreatePodTemplate(dsc.podTemplateController, ds, dsc.kubeClient)
+	if err != nil {
+		glog.Errorf("Couldn't get PodTemplate for daemon set %s: %v", ds.Name, err)
```

Why not return the error? If it's intended, it deserves a comment.